packages/atomic/src/components/common/facets/facet-common.tsx

```diff
@@ -1,5 +1,6 @@
 import {Schema, StringValue} from '@coveo/bueno';
 import {AnyFacetValuesCondition, AnyFacetValueRequest} from '@coveo/headless';
+import {SpecificSortCriteriaExplicitAlphanumeric} from '@coveo/headless/dist/definitions/features/facets/facet-api/request';
```

Comment:
That's a red flag, we should not be relying on types not exposed by Headless at a top level in Atomic.

It means the type is not properly exposed by Headless.

Secondly, (and more importantly) I don't think that's even something that should be leaking from Headless in the first place, or that we should need/reference in consumer projects (ie: Atomic).

My feeling is that the only thing we should expose from Headless is `FacetSortCriterion`, which is just the finite list of sort criterion.

And then internally in Headless, when we do receive `alphanumericDescending`, we map it to that type: `SpecificSortCriteriaExplicitAlphanumeric`.

In v3, if we want to only support those types of `Explicit` sort criteria, it would be a breaking change to manage.

Does it make sense ?